src/App.tsx

```diff
@@ -15,37 +18,96 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  name = 'name',
+  length = 'length',
+}
+
+const SORT_FIELD_NAME: SortField = SortField.name;
+const SORT_FIELD_LENGTH: SortField = SortField.length;
+
+function getPreparedGoods(
```

Comment:
The function's name 'getPreparedGoods' should start with a verb that clearly indicates what the function does, such as 'prepareGoods' or 'sortAndReverseGoods'.